Dockerfile

```diff
@@ -1,2 +1,4 @@
-FROM golang:1.5.1-onbuild
-EXPOSE 5000
\ No newline at end of file
+FROM golang:1.6.0-onbuild
+ADD . /app
```

Comment:
puts the code in `/app` path in the image so we can easily find it in the compose config